src/main/riddl/Order/Order.riddl

```diff
@@ -23,8 +23,7 @@ context OrderContext is {
   }
```

Comment:
At line 3 and line 22 are references to the OrderStatus type. @reid-spencer as per our talks yesterday on this topic, can you confirm that we want to remove something like this? If so, @JamesYopp I just wanted to let you know about this :)

Reply:
Removed everything status related from order.riddl assuming this is preferred.

Reply:
Okay.